README.md

```diff
@@ -1 +1,3 @@
-# rocketmq-client-nodejs
+# RocketMQ Client for Node.js
+
+It still under development.
```

Comment:
It would be helpful for community users to be involved in if we could make some comments on how to build from scratch.

Reply:
This PR is for several community volunteers who discussed last weekend. We will work together to finish the first available version.

Reply:
I've updated the README.md